invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -495,6 +495,12 @@ dl.details-list {
     align-items: flex-start;
     row-gap: 1rem;
   }
+
+  > *:nth-child(2) {
```

Comment:
does it make sense to apply in invenio-theme instead ?

Reply:
It should be part of the `.auto-column-grid` class, I think we only have this class in app-rdm?